examples/IMDb/README.md

```diff
@@ -0,0 +1,178 @@
+
+# Sentiment Analysis
+
+## The IMDB Dataset
+
+The **IMDB** dataset is one of the most popular sentiment analysis datasets. It contains multiple movie reviews, each annotated with either a `positive` or `negative` label.
+
+In this example, a classifier model was built to predict positive and negative reviews.
+
+### Text Preprocessing
+
+To preprocess the text, first lowercase all words and remove any HTML residues and punctuations. Tokenize the words and feed them into an **Embedding** layer to embed all the tokens.
+
+### Densely Connected Model
+
+A straightforward approach to solving this problem is by using a densely connected model. The proposed [model](https://github.com/tensorleap/tensorleap/blob/master/examples/imdb/imdb/imdb/model_infer.py) is composed from a `Dense` layer followed by a `GlobalAveragePooling1D`.
+
+Below is the model's block representation within our platform:
+
+![Dense Model (click-to-zoom)](images/img_1.png)
+
+#### Training Metrics
+
+During the training our platform collects various performance metrics and metadata that are presented interactively within a customizable Dashboard. This enables further performance and data analysis.\
+After training the model for 20 epochs, the **accuracy** is **0.9595** and the error **loss** is **0.149**. \
+Visualization of the error loss / accuracy vs batch:
+
+![Loss and Accuracy Metrics after Training](images/img.png)
+
+### Error Analysis
+
+#### Population Exploration
+
+The plot below is a **population exploration** plot. It represents a samples' similarity map based on the model's latent space, built using the extracted features from the trained model.
+
+The samples shown below were taken from the **training set** and are colored based on their **ground truth** class. The dot's size represents the model's error loss for that specific sample. We can see that there is a clear separation of the samples based on their classes.There are also failing samples (large dots).
+
+![Population Exploration Analysis of Training Samples](images/img_2.png)
+
+#### Unsupervised Clustering
+
+Tensorleap's platform provides an unsupervised clustering of samples based on their similarities.
+
+In our use-case we can see that this clustering technique was able to group together samples with common features. For example, in this Population Exploration plot, each cluster is colored, while the size of the dot represents the number accuracy each sample:
+
+![Population Exploration - Unsupervised Clustering](images/img_4.png)
+
+It can be seen that the left-side groups consist of samples with a known ground truth of "positive".
```

Comment:
Consider rephrasing the end. Positively labelled samples

Reply:
has changed